src/Properties/PhoneNumbers.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace GingerPluginSdk\Properties;
+
+use GingerPluginSdk\Bases\BaseField;
+use GingerPluginSdk\Collections\AbstractCollection;
+use GingerPluginSdk\Helpers\SingleFieldTrait;
+
+final class PhoneNumbers extends AbstractCollection
+{
+    use SingleFieldTrait;
+
+    public function __construct(string $number)
```

Comment:
`public function __construct(string... $numbers)` would allow you to pass all phone numbers at once

Reply:
Nice remark, thanks